README.md

```diff
@@ -128,6 +128,7 @@ eer
 - [SourceCodeObfuscator](https://github.com/0x000N3X4N/SourceCodeObfuscator) - C# source code obfuscator
 - [pshield](https://github.com/developervariety/pshield) - Plugin-based obfuscator
 - [PAOfuscator](https://github.com/Schaboom/PAOfuscator) - Obfuscator für Paradise.de
+- [MindLated](https://github.com/Sato-Isolated/MindLated) - .NET obfuscator that uses dnlib
```

Comment:
It seems to me that Isolated obfuscator has been renamed to MindLated. That's why it makes sense to remove the previous and leave the last one.

Reply:
Thanks !